client/types/KeyguardRequest.d.ts

```diff
@@ -140,9 +140,9 @@ declare namespace KeyguardRequest {
             // specifically used to trigger a redirect to create after returning to caller
             GOTO_CREATE: 'GOTO_CREATE',
             // used to signal a user initiated cancelation of the request
-            CANCEL: 'CANCEL',
+            CANCEL: 'CANCELED',
```

Comment:
The constant name should also be renamed, I'd say.

Reply:
Although, I'm not 100% sure. Because it also kind-of makes sense to treat is as the action (to cancel) and the result, that the user gets is "canceled".